### PR TITLE
HW1

### PR DESCRIPTION
**Pull request by sodic**

### CODE REVIEW

File: homework_1/task_1/fizzbuzz/fizzbuzz.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+			}
+		}
+
+		if sb.Len() != 0 {
+			return sb.String()
+		} else {
+			return strconv.Itoa(number)
+		}
+	}
+}
```
Comment (oreskdaxilis) [inline homework_1/task_1/fizzbuzz/fizzbuzz.go:50]: Generally I like the approach of treating mappers as first-class citizens, but instead of this JS-like freak you could just extract this inner function and call it directly from where you need it (`mapRange`). That would be better aligned with Golang coding style. Ok, in that case you don't have `dePeNdEncy iNjeCTion` kind of behaviour for `mapRange`, but then you should rather use structs with interface fields, which we didn't teach yet.
TLDR it is fine not to have dependency injection here in this homework, but since your approach also works I'm fine with it.

File: homework_1/task_2/main.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import (
+	"fmt"
+	"log"
+
+	"code-cadets-2021/homework_1/task_2/tax"
+)
+
+var taxBuckets = []tax.Bucket{
+	{
+		AppliedAbove: 1_000,
```
Comment (oreskdaxilis) [inline homework_1/task_2/main.go:12]: Where did you find this convention? I haven't seen it before in Go and Goland reports an error, even though code does work.

**Review by oreskdaxilis — APPROVED** [review]

File: homework_1/task_1/fizzbuzz/fizzbuzz.go (hunk 1 above)
Comment (sodic) [reply homework_1/task_1/fizzbuzz/fizzbuzz.go:50]: Lik mi je reviewo na engleskom, pa daj se ubij

Comment (sodic) [reply homework_1/task_1/fizzbuzz/fizzbuzz.go:50]: image

File: homework_1/task_2/main.go (hunk 2 above)
Comment (sodic) [reply homework_1/task_2/main.go:12]: Pajtončina je prvi uveo to, kao za readability. Tu je objasnjenje za go: https://golang.org/ref/spec#Integer_literals. Ne znam kaj Goland izbrijava, meni je okej. Mozda ti je konfiguriran na stariju verziju jezika?

Comment (oreskdaxilis) [reply homework_1/task_2/main.go:12]: English please.

> Mozda ti je konfiguriran na stariju verziju jezika?

Yep that could be the case, but I won't look into it. We don't usually write numbers like that (actually we almost never have numbers anywhere in code, as we define them in the application configuration, while in code we do various other stuff e.g. mapping models).

Comment (sodic) [reply homework_1/task_2/main.go:12]: Better to include the underscores right away than to later have to deal with the question:
> Nikola... Di su nule od stotice, a di od tisucice?

Comment (sodic) [reply homework_1/task_2/main.go:12]: > e.g. mapping models).

You're confusing "e.g." with "i.e."
